packages/form-js-editor/assets/form-js-editor-base.css

```diff
@@ -145,6 +148,60 @@
   visibility: visible;
 }
 
+.fjs-editor-container .fjs-drag-move {
+  padding-left: 3px;
+}
+
+.fjs-editor-container .fjs-field-resize-handle {
+  resize: horizontal;
+  position: absolute;
+  visibility: hidden;
+  width: 6px;
+  top: 9px;
+  bottom: 9px;
+  border: 1px solid var(--color-resize-handle-border);
+  border-radius: 50px;
+  background: var(--color-white);
+  z-index: 3;
+}
+
+.fjs-editor-container .fjs-field-resize-handle-right {
+  right: -4px;
+  cursor: ew-resize;
```

Comment:
Seems like Firefox has some problems with the (empty) drag preview image. I'll check whether we can do something against this.

<img width="1440" alt="Bildschirm­foto 2023-04-17 um 11 12 20" src="https://user-images.githubusercontent.com/9433996/232440103-3bc07ba5-ae9f-4c83-8a6f-dd03d29b6bf3.png">

Reply:
I fixed that via https://github.com/bpmn-io/form-js/pull/593/commits/59d464d0aa79ad46ad2351e3b045513c209f97e9 by using a `div` instead of a `canvas` element. Apparently, firefox has its problems with `canvas` 😄